CoinDock-api/routes/v1/users.php

```diff
@@ -23,6 +24,11 @@
     Route::get('logout', [UserController::class, 'logout']);
 });
 
-Route::get('/', [RecoveryKeyController::class, 'recoveyKey']);
+Route::get('/', [RecoveryKeyController::class, 'recoveryKey']);
 
 Route::get('/random' , [RecoveryKeyController::class, 'random']);
+
+Route::get('/users/{user}/signup/status/',[UserController::class,'signUpInfo'])
```

Comment:
route should be 
api/user/{userId}/signup
according to REST api standard.

When looking for status: the endpoint should return only a status property. Here we are returning a signup object representing the state of the user signup